Makefile

```diff
@@ -1,10 +1,10 @@
-ARDUINOSW_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
```

Comment:
None. It was probably inadvertent. It should be ?= as make should check if it is already set.